decentraland/unity-renderer
# feat: friend request polishment

## What does this PR change?
It contains all the UI polishments required.

## How to test the changes?
1. Go to: https://play.decentraland.zone/?renderer-branch=fix/friend-requests-refinement&kernel-branch=feat/receive-friend-request-rpc
2. Open the friend request list
3. Click any of the received requests
4. Reject the request. It should be removed from the list
5. Open another received request
6. Accept it
7. Click on any of the sent requests
8. Cancel it. It should be removed from the list
9. Make the peer user to accept your sent friend request. It should be removed from the list and the user should appear as a friend
10. Go to: https://play.decentraland.zone/?renderer-branch=fix/friend-requests-refinement&kernel-branch=feat/receive-friend-request-rpc
11. The friend requests should be displayed normally and all the interactions should be working as usual

## Our Code Review Standards
https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
